docs/publication-control.md

```diff
@@ -0,0 +1,58 @@
+# Publication control
+
+As mentioned in `OnlineBase` in our discussion about [helper models](helpers.md), anything that inherits from `OnlineBase` will have a manager that ensures that objects with `is_online == False` will be excluded from `Model.objects.all()`.
+For `OnlineBase` this is `cms.models.OnlineBaseManager`, for `PageBase` it is `cms.models.PageBaseManager`, etc.
+
+<aside>
```

Comment:
`<aside>` block here, I don't think this does anything unless it's only visible in docsify?

Reply:
This is how it renders in Docsify (not out-of-the-box, but with tweaks):

![Screenshot 2019-11-01 at 13 18 13](https://user-images.githubusercontent.com/4973097/68027372-16487d00-fcaa-11e9-9b5e-485d97d210cb.png)